rdtools/analysis_chains.py

```diff
@@ -956,6 +956,12 @@ def _clearsky_preprocess(self):
                 self.poa_global_clearsky,
                 pv_input="energy",
             )
+            warnings.warn(
+                """Clear-sky analysis is peformed but `power_expected` was passed in by user.
+                   In this case, the power normalization is not tied to the modeled clear-sky
+                   irradiance and the clear-sky workflow will provide the same results as
+                   the sensor workflow."""
+            )
```

Comment:
There should be some subtle differences in the filtering, so I suggest we modify the warning a bit to indicate results will be similar.
```suggestion
                """Clear-sky analysis is performed but `power_expected` was passed in by user.
                   In this case, the power normalization is not tied to the modeled clear-sky
                   irradiance and the clear-sky workflow may provide similar results to
                   the sensor workflow."""
            )
```

Reply:
@mdeceglie - yep, that sounds good to me.